apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -69,6 +73,26 @@ export default class GameLab extends P5Lab {
   }
 
   runButtonClick() {
+    // For AI Rubrics Pilot
+    if (
+      experiments.isEnabled('ai-rubrics') &&
+      hasScriptData('script[data-rubricdata]')
+    ) {
+      const rubricData = getScriptData('rubricdata');
+      const script = document.querySelector('script[data-level]');
+      const config = JSON.parse(script.dataset.level);
+      const teacherId = getStore().getState().currentUser.userId;
+      const {rubric, studentLevelInfo} = rubricData;
```

Comment:
I know this is a ton of variables - I went back and forth with just putting all this info right in the call to `sendEvent` but that looked real messy... again, would love style feedback here.

Reply:
I personally like the variables being pulled out here

Reply:
I will say, though, that this is more and more making me think we might want to move some of the rubric info into a redux store. We're effectively using the DOM as global state at this point

Reply:
@bethanyaconnor if you've got the vision for this, I'm wondering if you have any pointers to an example of a redux pattern we might want to follow? it's been a while since I've used redux and I forget whether there have been changes in best practices for how it is used.

Reply:
I don't know of any changes. The reason I say this is that whether or not a rubric is present on the page and whether or not it's the correct level are things needed in the teacher rubric view (`RubricContainer`), `TopInstructions`, and now here. In addition, we keep moving the state farther and farther up the chain of components within `RubricContainer`, so it seems like more places need more state.

I am not a huge fan of redux and doing any implementation is completely out of the scope of this PR, so I'm not completely suggesting we do anything yet. But needing to access the same information from the DOM as we access elsewhere feels like a hint that we might be heading in a direction where moving data to a global store makes sense.

Reply:
Thank you Bethany, this is very helpful! I agree that this should not block the current PR.